### PR TITLE
feat : added poster and info about project

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR contains

added poster and info about project

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @t-fine

**Comment by Ayush7614** [comment]
Cc @t-fine

**Comment by joewxboy** [comment]
@Ayush7614 Please link to the related issue, and prefix the PR title with `Issue-##: `

**Comment by Ayush7614** [comment]
> @Ayush7614 Please link to the related issue, and prefix the PR title with `Issue-##: `

@joewxboy Sorry I am not open any issue for this time for this PR but from next time i am always opening issue for the any changes and PR and please review and merged my code

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,14 @@
 # Welcome to Open-horizon-services 👋
+<p align="center">
+<img align="center" src="/profile/open-horizon-color.png" height="30%" width="30%"/>
+</p>
+
+Open-horizon-services mission is created house edge micro-services that can be easily built, tested, and published to any open horizon management hub.
+
+## About us
+
+An LF Edge open-source project that provides remote and autonomous management of containerized applications and related models that are deployed to distributed edge computing nodes and devices.
```
Comment (clementkng) [inline profile/README.md]: It might be clearer to say "This is an LF Edge..." to make it clear what we're referring to.

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,14 @@
 # Welcome to Open-horizon-services 👋
+<p align="center">
+<img align="center" src="/profile/open-horizon-color.png" height="30%" width="30%"/>
```
Comment (clementkng) [inline profile/README.md]: This won't need be addressed for this review, but we've copied the `open-horizon-color.png` in a lot of places for the logo. I wonder if we should try to load it from the https://github.com/open-horizon/artwork repo instead.

**Comment by Ayush7614** [comment]
cc @clementkng @t-fine

**Review by clementkng — APPROVED** [review]

**Review by joewxboy — APPROVED** [review]

**Pull request merged**